types/3box/index.d.ts

```diff
@@ -0,0 +1,150 @@
+// Type definitions for 3box 1.22
+// Project: https://github.com/3box/3box-js#readme
+// Definitions by: Shikanime Deva <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/3box/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me